TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ReservationDao.java

```diff
@@ -0,0 +1,31 @@
+package cz.muni.fi.travelAgency.dao;
+import cz.muni.fi.travelAgency.entities.Customer;
+import cz.muni.fi.travelAgency.entities.Excursion;
+import cz.muni.fi.travelAgency.entities.Reservation;
+import cz.muni.fi.travelAgency.entities.Trip;
+
+import java.util.Date;
+import java.util.List;
+
+public interface ReservationDao {
```

Comment:
Inerface should also contain java doc for of the methods. Please add them in your next commit